_literate/EX-boston-flux.jl

```diff
@@ -0,0 +1,128 @@
+# **Main author**: Ayush Shridhar (ayush-1506).
+#
+# ## Getting started
+
+using MLJFlux, MLJ, DataFrames, Statistics, Flux
```

Comment:
please `import` specific names from `DataFrames` to avoid name clashes with MLJ

also you seem to use qualified imports from Flux and MLJFlux so might as well just `import` them instead of `using`

Reply:
You need `using Random` here.